fearless/Common/Extension/Model/ConnectionItem+Default.swift

```diff
@@ -4,22 +4,22 @@ import IrohaCrypto
 extension ConnectionItem {
     static var defaultConnection: ConnectionItem {
```

Comment:
Do we still need that? @ERussel 
Before we had a Kusama account as a default, thus this was probably used, but now users can create an account in any network as their first account, so maybe we can just use supportedConnections?

Reply:
Yep, let's leave as it is for now